python/ray/air/_internal/remote_storage.py

```diff
@@ -136,6 +136,36 @@ def is_non_local_path_uri(uri: str) -> bool:
 _cached_fs = {}
 
 
+def is_mounted(path: str) -> bool:
```

Comment:
@krfricke could you double check this logic? Is this generalizable?

Reply:
This thing checks if one of your parents is a mount point. Labelling all of these as "nfs" is not very accurate, but I couldn't figure out a very good way to detect network mount vs. a non-network mount. Saving to a USB drive would also be mounted in this case...

I'm okay with just removing this check. Just consider it as part of local + no sync.

Reply:
Let's at least rename this to `mount`? It's quite common to have a `/home` directory mounted to a different disk, so on local machines this would definitely be imprecise.

Is there any way to detect network mounts as opposed to disk mounts?

Reply:
I searched how to do this but couldn't figure it out... But I just asked chatgpt at apparently `psutil.disk_partitions` exposes a `fstype` property that can show network vs disk mount. I'll try that out!